reinstall.sh

```diff
@@ -40,4 +40,7 @@ if [ -x "$(command -v conda)" ]; then
   # conda update -c numba numba -y >  /dev/null 2>&1 && echo "Numba updated!" || echo "Numba could not be updated!"
 fi
 
+echo 'Installing k2 FSA framework'
```

Comment:
What is the installation time for this? If its too long, we need to avoid doing this on normal CI and await Blossom CI support.

Reply:
The installation time is ~2 min on my workstation and ~15 min on CI. Removed from here in [f0308a1](https://github.com/NVIDIA/NeMo/pull/3351/commits/f0308a1a8e24f93b1ed3e00501a7a5ef1a6f4e55)